internal/proto/controller/storage/servers/store/v1/worker.proto

```diff
@@ -45,7 +45,7 @@ message Worker {
 // worker_id, key, value, and source.
 message WorkerTag {
   // worker_id is the public key that key of the worker this tag is for.
-  // @inject_tag: `gorm:"default:null"`
+  // @inject_tag: `gorm:"primary_key"`
```

Comment:
I think the primary_key tag needs to also be added on key and value to represent the composite primary key for worker tags